openmc/model/model.py

```diff
@@ -235,6 +236,12 @@ def init_lib(self, threads=None, geometry_debug=False, restart_file=None,
             MPI intracommunicator
         """
 
+        try:
+            import openmc.lib
+        except ImportError:
+            raise ImportError('openmc.lib must be available to '
+                'execute Model.init_lib(...)') from None
```

Comment:
Perhaps something like this then

```suggestion
        import openmc.lib
```